### PR TITLE
Add new rule operator-linebreak

### PR DESCRIPTION
**Pull request by rowbi-wan**

### CODE REVIEW

File: packages/eslint-config/rules/core/operator-linebreak/correct.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+foo = 1 + 2;
+
+foo = 1
+    + 2;
+
+foo
+    = 5;
```
Comment (samboylett) [inline packages/eslint-config/rules/core/operator-linebreak/correct.js]: Not a fan of this but the rest looks ok

Comment (BeccaAnderton) [reply packages/eslint-config/rules/core/operator-linebreak/correct.js]: Agree with the above

Comment (rowbi-wan) [reply packages/eslint-config/rules/core/operator-linebreak/correct.js]: was hoping I'd be able to add a rule override for single operators being inline but cant see it anywhere.

**Comment by samturrell** [comment]
Sorry @rowbi-wan

**Pull request merged**